File: src/main/java/me/sul/worldmanager/WorldManager.java
package me.sul.worldmanager;

import me.sul.worldmanager.summonmob.AutomaticallySummonMob;
import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.bukkit.plugin.java.JavaPlugin;

public final class WorldManager extends JavaPlugin {
    private static WorldManager instance;

    @Override
    public void onEnable() {
        instance = this;
        new AutomaticallySummonMob();
    }

    @Override
    public void onDisable() {

    }

    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {

        return true;
    }

    public static WorldManager getInstance() { return instance; }
}

File: src/main/kotlin/kr/sul/worldmanager/WorldManager.kt
package kr.sul.worldmanager

import kr.sul.worldmanager.setting.Setting
import kr.sul.worldmanager.summonmob.AutomaticallySummonMob
import org.bukkit.plugin.java.JavaPlugin

class WorldManager : JavaPlugin() {
    companion object {
        lateinit var instance: WorldManager private set
    }

    override fun onEnable() {
        instance = this
        AutomaticallySummonMob(instance)
        Setting(instance)
    }

    override fun onDisable() {}
}